addons/project/migrations/13.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,40 @@
+---Models in module 'project'---
+---Fields in module 'project'---
+project      / account.analytic.account / project_count (integer)       : previously in module hr_timesheet
+project      / account.analytic.account / project_ids (one2many)        : previously in module hr_timesheet
+project      / project.project          / analytic_account_id (many2one): previously in module hr_timesheet
+project      / project.project          / percentage_satisfaction_project (integer): DEL
+project      / project.project          / percentage_satisfaction_task (integer): DEL
+project      / project.project          / rating_ids (one2many)         : NEW relation: rating.rating
+# NOTHING TO DO
+project      / project.task             / company_id (many2one)         : now required, req_default: function
+# DONE: post-migration: filled the field when empty
+project      / project.task             / date_start (datetime)         : DEL
+project      / project.task             / notes (text)                  : DEL
+project      / project.task.type        / legend_priority (char)        : DEL
+# NOTHING TO DO
+
+---XML records in module 'project'---
+NEW ir.actions.act_window: project.portal_share_action
+NEW ir.model.access: project.access_account_analytic_account_manager [renamed from hr_timesheet module]
+NEW ir.model.access: project.access_account_analytic_account_user [renamed from hr_timesheet module]
+NEW ir.model.access: project.access_account_analytic_line_project [renamed from hr_timesheet module]
+# NOTHING TO DO: Handels in hr_timesheet
```

Comment:
no, you have to handle the xmlid renames in this module.

Reply:
it's `renamed from` so might be not exist. i guess they should rename when it's `renamed to`

Reply:
@Yasaie That is what @MiquelRForgeFlow is saying. You need to rename it pre-migration if it already exists. Otherwise it will create new ones. The `rename_xmlids` will take care if it doesn't exist.

Reply:
@gdgellatly I mean in case of `DEL` of xml we need to do actions like `rename_xmlids` otherwise if it's already renamed in `hr_timesheet` nothing to do unless it will create a new one.
I think this more correct way

Reply:
@Yasaie I think you are misunderstanding what this means because it is obvious you must handle it here.

hr_timesheet depends on project. The xmlid has moved from hr_timesheet to project. If you do not rename it here pre-migration and hr_timesheet is installed you will duplicate.